src/libsync/propagateupload.cpp

```diff
@@ -21,6 +22,9 @@
 #include "utility.h"
 #include "filesystem.h"
 #include "propagatorjobs.h"
+#include "transmissionchecksumvalidator.h"
+#include "configfile.h"
```

Comment:
Actually configfile.h can be removed here, but it is (still) used in the transmissionchecksumvalidator. Which is ... not much better I suppose?